kubejs/server_scripts/enigmatica/kubejs/base/recipes/remove.js

```diff
@@ -34,6 +34,14 @@ events.listen('recipes', function (event) {
         'quantumstorage:qsu'
     ];
 
+    materialsToUnify.forEach((material) => {
+        if (material == 'compressed_iron') {
+            outputRemovals.push('pneumaticcraft:' + material + '_gear');
+        } else {
+            outputRemovals.push('emendatusenigmatica:' + material + '_gear');
+        }
+    });
+

```

Comment:
To reduce the need for looping, this should be done within the function that re-adds the recipes, although it's not as pretty as putting it here :)

Reply:
Also, it could be done with:

```js
 event.remove({
        output: '/\\w+:\\w+_gear/',
        type: 'minecraft:crafting_shaped'
    });
    ```